utils/list_action.rb

```diff
@@ -0,0 +1,30 @@
+require_relative 'fetchers/fetcher'
+
+module Util
+  class ListAction < Trello::BasicData
+    extend Util::Fetcher
```

Comment:
I'm not a fan of this being both BasicData and a Fetcher. The current design was created to purposefully limit the fetchers to doing only one thing: accessing the Trello API through http calls.